h/services/delete_user.py

```diff
@@ -63,6 +62,11 @@ def _delete_groups(self, groups):
         for group in groups:
             self.request.db.delete(group)
 
+    def _unassign_groups_creator(self, groups):
+        for group in groups:
+            group.creator = None
+        self.request.db.flush()
```

Comment:
Was there a particular reason for the `flush` call here? The changes will be flushed to the DB automatically either at the end of the request when the transaction is committed, or when the next query happens (see [autoflush docs](https://docs.sqlalchemy.org/en/latest/orm/session_api.html?highlight=autoflush#sqlalchemy.orm.session.Session.params.autoflush)). This is why there are relatively few explicit calls to `db.flush` in our code, and those are mainly in cases where we want to force any errors that would occur during the flush to happen at a specific point rather than whenever SQLAlchemy deems a flush is necessary.

Reply:
No, I originally thought I needed it but turns out it was something else and I just forgot to remove it.

Reply:
Durp-think I got it this time.